generators/python-v2/codegen/src/ast/PythonFile.ts

```diff
@@ -102,7 +102,7 @@ export class PythonFile extends AstNode {
                 if (levelsUp > 0) {
                     relativePath += ".";
                 }
-                relativePath += refModulePath.slice(commonPrefixLength).join(".");
+                relativePath += refModulePath.slice(commonPrefixLength + 1).join(".");
```

Comment:
Might be worth leaving a comment explaining why we add one. I assume it's because we assume it's under the same module root?